src/pages/settings/support.vue

```diff
@@ -0,0 +1,190 @@
+            </div>
+            <div class="mt-12">
+              <a
+                class="text-like-green font-500"
+                :href="getStripeBillingPortalAPI"
+              >{{ $t('SettingsSupportPage.BillingHistory') }}</a>
+            </div>
+          </div>
+        </div>
+        <div class="mt-20">
+          <div class="font-500">{{ getUserSubscriptionInfo.currentPeriodEndString }}</div>
+          <div>{{ $t('SettingsCivicPage.billingSummary.nextBillingDate') }}</div>
+        </div>
+      </div>
+    </template>
+
+    <h2
+      class="mt-40 text-like-green font-24 font-500"
+    >{{ $t('SettingsSupportPage.Title.ManageSupportingUser') }}</h2>
+    <ul
+      v-if="supportingLikerIds.length"
+      key="content"
+      class="supporting-liker-list mt-12"
+    >
+      <li
+        v-for="id in supportingLikerIds"
+        :key="id"
+      >
+        <SupportingLikerView
+          class="h-full"
+          :liker-id="id"
+          :price="getAuthorQuantity(id) * 5"
+        />
+      </li>
+    </ul>
+    <div
+      v-else
+      class="bg-white rounded-8 mt-20 px-32 py-24 text-center text-16 font-500 text-gray-4a"
+    >{{ $t('SettingsSupportPage.NoSupport') }}</div>
+
+    <NuxtChild />
+  </div>
+</template>
+<script>
+import { mapActions, mapGetters } from 'vuex';
+
+import Spinner from '~/components/Spinner/Spinner';
+import SupportingLikerView from '~/components/SupportingLikerView/SupportingLikerView';
+
+import { getStripeBillingPortalAPI } from '~/util/api';
+
+function getMaskedCardNumber(brand, last4) {
+  switch (brand) {
+    case 'visa':
+    case 'mastercard':
+      return `•••• •••• •••• ${last4}`;
+
+    case 'amex':
+      return `•••• •••••• •${last4}`;
+
+    default:
+      return `•••• ${last4}`;
+  }
+}
+
+export default {
+  components: {
+    SupportingLikerView,
+    Spinner,
+  },
+  data() {
+    return {
+      state: 'loading',
+    };
+  },
+  computed: {
+    ...mapGetters([
+      'getUserSubscriptionInfo',
+      'getUserIsCivicLiker',
+      'getCivicSupportingUsers',
+    ]),
+    getStripeBillingPortalAPI,
+    supportingLikerIds() {
+      return Object.keys(this.getCivicSupportingUsers);
+    },
+    maskedCardNumber() {
+      if (this.getUserSubscriptionInfo) {
+        const {
+          card: { brand, last4 },
+        } = this.getUserSubscriptionInfo;
+        return getMaskedCardNumber(brand, last4);
+      }
+      return '';
+    },
+  },
+  async mounted() {
+    if (!this.getUserIsCivicLiker) {
+      this.$router.push({
+        name: 'settings-support-users-id',
+        params: { id: 'foundation' },
+      });
+      this.state = 'error-not-civic';
+      return;
+    }
+    try {
+      await Promise.all([
+        this.fetchCivicSupportingUsers(),
+        this.fetchUserSubscriptionInfo(),
+      ]);
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(err);
+      this.state = 'error-civic-v2';
+    }
+    this.state = 'done';
```

Comment:
Does this make `error-civic-v2` always get replaced by `done`?

Reply:
I blame you